chapters/1.md

```diff
@@ -1,100 +1,185 @@
+[Figure X.3]
+## Principles of Monero
 
-## Why open source?
+Monero is designed with the following principles in mind:
 
-Satoshi Nakimoto created the blockchain with a goal to have people place trust in each other rather than in centralized third parties. Open-source software supports this goal, as anyone can review the code to make sure that it is secure. For example, the Linux kernel is open-source and has over 5,062 contributors, and it is widely considered to be a highly secure operating system.
+* **Network Decentralization**: The Monero network and ledger are globally distributed. This means that there is no single server or database that can be maliciously controlled or censored. If one government were to shut down Monero nodes in their country, or attempt to limit who can send/receive Monero, the effort would be in vain! The rest of the world will carry on the network and process any valid transactions.
 
-The benefits of open-source software, namely trust and security, align well with the goals of cryptocurrency, which is why the code to The Monero Project is completely open-source.
+* Financial Security: The globally-distributed Monero has no central weak point that can be hacked to steal your funds. Your Moneroj are secured by incorrigible cryptographic techniques, so there is no need to trust a third party with responsibility over your funds. Every single Monero participant can verify the validity of the ledger themselves, so you do not even have to trust the node operators! (You can learn more about the cryptographic techniques that secure Monero in chapter X)
+Financial Privacy: ​Most blockchain projects increase security at the expensive of privacy, however Monero prioritizes your privacy. Transaction amounts, sender identity, and recipient identity are all obfuscated on the blockchain, so your Moneroj storage and spending activities are not trackable.
 
-The Monero Project primarily uses Git for version control (more information for developers in the 5th chapter), which also makes it easier for people to view every single proposed line of code to be added, removed or modified.
+* Fungibility: ​The term “fungibility” means that some type of asset is considered interchangeable. For example, imagine that you let your neighbor borrow 1 kilogram of flour for a cake. When they return flour the next week, of course it will be 1 kilogram of flour from a different source (since they used your original flour for baking). This is not a problem, since flour is fungible. However, vehicles are not fungible; if you let your neighbor borrow your car, you probably want the same one back!
```

Comment:
I honestly think the flour example is terrible lol... you should probably just liken it to fiat money or gold of equal weight.

Reply:
I have some doubts about this section, we have to improve it